services/movie.py

```diff
@@ -0,0 +1,34 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet | Movie:
```

Comment:
It returns only QuerySet
```suggestion
               actors_ids: list[int] = None) -> QuerySet[Movie]:
```